test/e2e/simple_test.go

```diff
@@ -0,0 +1,55 @@
+package e2e
+
+import (
+	"context"
+	"errors"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/test/txsim"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	latestVersion = "v1.0.0-rc9"
```

Comment:
So this e2e will not run latest master, but instead the specified release branch?

Reply:
Yeah, so far it has to retrieve the image from github (i.e. the docker image needs to be published first). It would be nice if we could just build the image for the current branch locally as a means of testing against. Something to follow up with later

Reply:
Cool. I guess it would make sense to create an issue for this as an improvement